packages/context/src/__tests__/acceptance/class-level-bindings.acceptance.ts

```diff
@@ -186,7 +186,7 @@ describe('Context bindings - Injecting dependencies of classes', () => {
 
     ctx.bind(STORE_KEY).toClass(Store);
     expect(() => ctx.getSync<Store>(STORE_KEY)).to.throw(
-      'The type of Store.constructor[0] (String) is not a Getter function',
+      'The type of Store.constructor[0] (String) is not Getter function',
```

Comment:
Is there any particular reason for removing the article `a`? English is not my mother tongue, but it seems to me that "a Getter function" is more correct grammatically.

(Same comment applies for the second test changed below.)

Reply:
IMO, `a` hints an instance, not the type/class. I removed `a` to be consistent with other types, such as `Array` or `ContextView`.